src/index.html

```diff
@@ -1,17 +1,165 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <label class="form_field">
+          Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            maxlength="30" 
+            autocomplete="off"
+           required
+          > 
+        </label>
+        
+        <label class="form_field">
+          Name:
+          <input 
+            type="text" 
+            name="Name" 
+            autocomplete="off"
+            required
+          >
+        </label>
+        
+        <label class="form_field">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+        
+        <label class="form_field">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+       
+        <label class="form_field">
+          I accept the term of the agreement
+          <input type="checkbox" name="accept">
+        </label>
+        
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+
+        <label class="form_field">
+          E-mail:
+          <input 
+            type="email" 
+            name="e-mail" 
+            placeholder="email@example.com"
+          >
+        </label>
+        
+        <label class="form_field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="4" 
+            maxlength="12"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form_field">
+          Do you love cats?
+          <input 
+            type="radio" 
+            name="yes or no" 

```

Comment:
the form should distinguish the fields by name, if they are named the same, then the data will be invalid when the form is saved

```suggestion
            name="yes" 

```